frameworks/template/src/main/resources/logback.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file doesn't need to exist, nothing uses it. Populate `simplelogger.properties` (see [here](https://www.slf4j.org/api/org/slf4j/impl/SimpleLogger.html)) until we move to a real logging backend.